### PR TITLE
RCE 'textButtons' prop removed in benefit of config.getToolbarSettings; plugin text buttons settings added

### PR DESCRIPTION
**Pull request by lxgreen**

### CODE REVIEW

File: packages/editor/src/RichContentEditor/Toolbars/default-toolbar-settings.js [hunk 1]
```diff
@@ -15,155 +16,199 @@ const defaultSideToolbarVisibilityFn = editorState => {
...
+          }
+        };
+      },
+      getPositionOffset: () => ({
+        desktop: { x: -40, y: 0 },
         mobile: {
-          ios: shouldCreate,
-          android: shouldCreate,
+          ios: { x: 0, y: 0 },
+          android: { x: 0, y: 50 },
```
Comment (bchmn) [inline packages/editor/src/RichContentEditor/Toolbars/default-toolbar-settings.js]: this should be { x: 0, y: 0 }

File: docs/ToolbarCustomization.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ The `getToolbarSettings` is defined as follows: `{ textButtons, pluginButtons }
 The `Setting` type is defined as follows:
 ```
 {
-  name: TOOBAR_TYPE,
+  name: TOOBARS.TYPE,
```
[the remark this answers is not in the document]
Comment (lxgreen) [reply docs/ToolbarCustomization.md]: it is TOOBARS.TYPE, isn't it?? :)

[the remark this answers is not in the document]
Comment (lxgreen) [reply docs/ToolbarCustomization.md]: good catch!!

File: packages/editor/src/RichContentEditor/Toolbars/default-toolbar-settings.js (hunk 1 above)
Comment (lxgreen) [reply packages/editor/src/RichContentEditor/Toolbars/default-toolbar-settings.js]: it is ok

Comment (bchmn) [reply packages/editor/src/RichContentEditor/Toolbars/default-toolbar-settings.js]: I doubt it, I changed it today
https://github.com/wix-incubator/rich-content/commit/fe13b6daa5327306f858fda79b1faa10e49eab9d

**Pull request merged**